pkg/scheduler/plugins/priority/priority.go

```diff
@@ -113,6 +113,26 @@ func (pp *priorityPlugin) OnSessionOpen(ssn *framework.Session) {
 	}
 
 	ssn.AddPreemptableFn(pp.Name(), preemptableFn)
+
+	validJobFn := func(obj interface{}) *api.ValidateResult {
+		_, ok := obj.(*api.JobInfo)
+		if !ok {
+			return &api.ValidateResult{
+				Pass:    false,
+				Message: fmt.Sprintf("Failed to convert <%v> to *JobInfo", obj),
+			}
+		}
+
+		return nil
+	}
+	ssn.AddJobValidFn(pp.Name(), validJobFn)
+
+	jobStarvingFn := func(obj interface{}) bool {
+		ji := obj.(*api.JobInfo)
+		return ji.ReadyTaskNum() < int32(len(ji.Tasks))
```

Comment:
Please take a look whether we need to consider the WaitingTaskNum here as well?

Reply:
it is the same as the code in gang plugin。 it is repeat。 I think it is better to modify the default value of the function JobStarving and change from false to true

Reply:
> Please take a look whether we need to consider the WaitingTaskNum here as well?

Yes, It is needed. Let me add it.

Reply:
> it is the same as the code in gang plugin。 it is repeat。 I think it is better to modify the default value of the function JobStarving and change from false to true

No, they are not same. `minAvailable` is not considered here.